koku/api/tags/view.py

```diff
@@ -80,7 +81,6 @@ def get(self, request, **kwargs):
 
     def validate_key(self, key):
         """Validate that tag key exists."""
-        for handler in self.tag_handler:
-            if handler.objects.filter(key=key).exists():
-                return True
+        if EnabledTagKeys.objects.filter(provider_type__in=self.tag_providers, key=key).exists():
+            return True
         return False
```

Comment:
You can remove this branching:
```suggestion
        return EnabledTagKeys.objects.filter(provider_type__in=self.tag_providers, key=key).exists()
```